alphatwirl/concurrently/TaskPackageDropbox.py

```diff
@@ -93,6 +100,8 @@ def _collect_pkgidx_result_pairs_of_finished_tasks(self):
             logger = logging.getLogger(__name__)
             logger.warning('resubmitting {}'.format(self.workingArea.package_path(pkgidx)))
 
+            try: self.dispatcher.walltime = self.dispatcher.walltime*2
```

Comment:
Why do you need to double the wall time?  Is it worth adding a comment here to make it clear why you do it for a future developer?

Reply:
Kind of arbitrary to double it. But, at least at Imperial, if a job fails it might well be due to hitting the wall time. To save a continuous re-submission hitting the wall time each time I thought I'd increase it. If the number of jobs failing is minimal, as it hopefully should be, it shouldn't impact the job queuing too much.

I can add a comment saying this or change it.

Reply:
I see.  That sounds like a reasonable thing to do to me.  Might be nice to be able to disable this easily though.  So perhaps could you:
1. Add an option to `__init__` that enables / disables this behaviour (enabled by default)
2. Add a comment above here as to what it does and why?

How many times is a job resubmitted before it's truly marked as `failed` here?